src/rai_asr/rai_asr/asr_node.py

```diff
@@ -135,11 +226,21 @@ def _setup_publishers_and_subscribers(self):
             callback_group=self.callback_group,
         )
 
-    def _load_whisper_model(self):
-        self.openai_client = OpenAI()
-        self.model = partial(
-            self.openai_client.audio.transcriptions.create, model=self.whisper_model
-        )
+    def _initialize_asr_model(self):
+        if self.model_vendor == "openai":
+            from rai_asr.asr_clients import OpenAIWhisper
+
+            self.model = OpenAIWhisper(
+                self.model_name, DEFAULT_SAMPLING_RATE, self.language
+            )
+        elif self.model_vendor == "whisper":
+            from rai_asr.asr_clients import LocalWhisper
+
+            self.model = LocalWhisper(
+                self.model_name, DEFAULT_SAMPLING_RATE, self.language
```

Comment:
`DEFAULT` part in `DEFAULT_SAMPLING_RATE` suggests that this value can be changed, but it doesn't. I would name it `SAMPLING_RATE` or allow parametrization if that's the case

Reply:
Renamed to `VAD_SAMPLING_RATE`